client/app/components/_pickupList/pickupListItem/pickupListItem.styl

```diff
@@ -0,0 +1,54 @@
+a
+    color: inherit
+    text-decoration: none
+    margin-left: .5em
+
+img
+    width: 2em
+    height: 2em
+    border-radius: 50%
```

Comment:
For that, we would need something like https://github.com/css-modules/css-modules (I tried it out in a toy project and it worked nicely). We could look into it in the future, not sure how it integrates with angular.